apps/web/package.json

```diff
@@ -85,7 +85,7 @@
 		"embla-carousel-react": "^8.0.0-rc11",
 		"emoji-mart": "^5.5.2",
 		"eslint-config-next": "^14.0.4",
-		"firebase": "8.3.3",
+		"firebase": "10.9.0",
```

Comment:
@NdekoCode / @CREDO23 any idea why we refer to Firebase!? I don't think it's used... Can you check please??

Reply:
@evereq, the Firebase module is used here:  https://github.com/ever-co/ever-teams/blob/develop/apps/web/lib/features/integrations/boards/firebase.ts